app/views/health_cards/show.pdf.erb

```diff
@@ -55,8 +55,16 @@
         <% end %>
       </div>
     </div>
-    <div id="qr-code"></div>
-    <img src = '<%=@image_uri%>' >
-
+    <div id="qr-code">
+      <div class="tile is-ancestor">
+        <% @exporter.qr_codes.chunks.each do |ch| %>
+          <div class="tile is-parent">
+            <div class="tile is-child box has-text-centered">
+              <%= wicked_pdf_image_tag patient_qr_code_url(@patient, ch.ordinal, format: :png) %>
```

Comment:
I shrunk the QR code and it can now support up to 3 immunizations without paging. We may want to consider adding a task to redesign the PDF as each immunization takes up a lot of space, and I'm not sure if we should have all the patient data displayed either.

Reply:
I agree. [DVCI-98](https://jira.mitre.org/browse/DVCI-98) should take care of this. @ssayer feel free to add anything to the ticket that you think should be modified with the patient data.